sdk/python/feast/field.py

```diff
@@ -25,31 +27,43 @@ class Field:
     Attributes:
         name: The name of the field.
         dtype: The type of the field, such as string or float.
+        tags: User-defined metadata in dictionary form.
     """
 
     name: str
     dtype: FeastType
+    tags: Dict[str, str]
 
     def __init__(
-        self, *, name: str, dtype: FeastType,
+        self,
+        *,
+        name: Optional[str] = None,
```

Comment:
can we leave this as required (as well as `dtype`) but make `tags` optional?

Reply:
Sure.